tests/integration/test_user_flows.py

```diff
@@ -39,21 +39,22 @@ def test_confirm_email_address(self, client, email, idp, name):
         assert resp.status_code == 302
         assert resp.location.endswith(url_for('main.confirm_dept'))
 
+    @pytest.mark.xfail
```

Comment:
These tests need to be rewritten, but it will take a while longer - I marked them as xfail as a reminder.